src/derive/curve.rs

```diff
@@ -389,15 +399,33 @@ macro_rules! new_curve_impl {
         }
 
 
+        impl $name_jac_ext {
+            pub fn generator() -> Self {
+                let generator = $name_affine::generator();
+                $name_jac_ext::from(generator)
+            }
+
+            pub const fn curve_constant_a() -> $base {
+                $name_affine::curve_constant_a()
+            }
+
+            pub const fn curve_constant_b() -> $base {
+                $name_affine::curve_constant_b()
+            }
+        }
+
+
 
         // Jacobian implementations
 
+        // From affine to homogeneous
```

Comment:
Agreed.  I added the comment because I found it confusing too, since `$name` refers to homogeneous coordinates and `$name_affine` to affine coordinates.  Alternatively, I can remove the first comment, since non-extended Jacobian coordinates should be gone anyway.